Formula/enigma.rb

```diff
@@ -7,7 +7,8 @@ class Enigma < Formula
   revision 4
 
   livecheck do
-    url :stable
+    url :homepage
+    regex(/v?(\d+(?:\.\d+)+) Released/i)
```

Comment:
```suggestion
    regex(%r{url=.*?/enigma[._-]v?(\d+(?:\.\d+)+)\.t}i)
```

Reply:
you are right, I was thinking the page_match, this regex makes sense from the artifact perspective. 👍